haystack/utils/openai_utils.py

```diff
@@ -6,7 +6,7 @@
 import json
 from typing import Dict, Union, Tuple, Optional, List
 import requests
-from tenacity import retry, retry_if_exception_type, wait_exponential, stop_after_attempt
+import tenacity as tc
```

Comment:
```suggestion
import tenacity
```

Let's not create indirection.